gui/src/utils/sessionStorage.js

```diff
@@ -0,0 +1,5 @@
+import * as storage from 'utils/storage'
+
+export const get = () => storage.get('session')
```

Comment:
Good call on the filename. I was using the `session` attribute from redux state but I can see how that can be confusing given that there is a browser api for `window.sessionStorage`. How do you feel about renaming the reducer to something like:

```
{ 
   // session: {...}
   authentication: {
      // authorized: true ->
      allowed: true
   }
}
```

I perhaps extracted too much, too early... But I'm envisioning `utils/storage.js` as helper methods to `get/set` storage values under a `chainlink` namespace and handle edge cases like invalid JSON. `utils/sessionStorage.js` is just a convenience helper to avoid having to remember the magic key `session`.

Reply:
Good point. Sure, let's change the name of the reducer to authentication or whatever you come up with.